test/alternator/test_ttl.py

```diff
@@ -702,7 +729,7 @@ def read_entire_stream(dynamodbstreams, table):
 # in more than one page - so verifies that the expiration scanner service
 # does the paging properly.
 @pytest.mark.veryslow
```

Comment:
I think the veryslow marker above should be dropped too

Reply:
You're right, it's not like other tests which need to wait until a timeout. But it's slow because it creates a lot of data (40,000 items), and takes almost 5 seconds. I needed a lot of data to make sure the paging passed 1MB (and when I set this length, I checked which one reproduced the bug).

I'm still so in love with the idea that test/alternator/run takes 20 seconds to run all tests (almost 700 of them!) that it's hard for me to consider the fact that a single test takes 5 seconds as "normal" ;-) But maybe I need to come up with a different classication - it's not a "veryslow" test - it's a "somewhatslow" test, and even if developers won't run these, Jenkins can (if it spend hours on other tests, increasing Alternator tests from 20 to 25 seconds shouldn't matter).